assemblies/assembly-openshift-authoring-managed.adoc

```diff
@@ -17,12 +17,10 @@ and other business assets.
 .Prerequisites
 
 * {OPENSHIFT} version 3.11 is deployed.
-* At least four gigabytes of memory are available in the OpenShift cluster/namespace.
-* For a high-availability deployment, the following resources are available on the OpenShift cluster:
-** For the {CENTRAL} replicated pod, 8 gigabytes of memory and 2 CPU cores are required for each replica. Two replicas are created by default.
-** For the {KIE_SERVER} replicated pod, 1 gigabyte of memory and 1 CPU core are required for each replica. Two replicas are created by default.
-** The Red Hat AMQ replicated pod uses the default resource limits configured on your cluster.
-** For the Red Hat Data Grid replicated pod, 2 gigabytes of memory and 1 CPU core are required for each replica. Two replicas are created by default.
+* The following resources are available on the OpenShift cluster. Depending on the load, higher resource allocation might be necessary for acceptable performance.
+** For the {CENTRAL} pod, 4 gigabytes of memory and 2 virtual CPU cores. In a high-availability deployment, these resources are required for each replica and two replicas are created by default.
```

Comment:
Minor rewrite suggestion for both bullets:

, these resources are required for each replica. Two replicas are created by default.